app/views/partials/_management_details.html.erb

```diff
@@ -0,0 +1,17 @@
+<% total = 0 %>
+<% @purchase.each { |num| total += num[:amount] } %>
+<ul class="details-wrapper d-flex flex-column">
+  <li class="d-flex details-category border-details">
+    <div class="icon">
+      <%= image_tag("#{@category.icon}", alt: "#{@category.name} icon") %>
+    </div>
+    <p class="detail-name"><strong><%= @category.name %></strong></p>
+    <p class="cat-amount">Total spent: $<%= total %></p>
+  </li>
+  <% @purchase.each do |good| %>
+    <li class="d-flex border-details f-justify-between details-cards">
+      <p><%= good[:name] %></p>
+      <p>$<%= good[:amount] %></p>
+    </li>
+  <% end %>
+</ul>
```

Comment:
As part of the project requirement, the list of transactions should be in order of the most recent

![image](https://user-images.githubusercontent.com/66112760/159091057-cee37d71-a869-44bd-af88-f1c748175ec8.png)

Kindly fix this 👍

Reply:
Hello @bimbolabuari thanks for your feedback, I've already fixed it! 🚀